test/validation/api/Makefile.am

```diff
@@ -32,7 +32,7 @@ SUBDIRS = $(ODP_MODULES)
 include $(top_srcdir)/test/Makefile.inc
 TESTS_ENVIRONMENT += TEST_DIR=${top_builddir}/test/validation
 
-TESTS = \
+TESTS_PARALLEL = \
```

Comment:
The way to avoid parallel make check is simply to not use the -j option. Please note that this series does not introduce parallelism for validation tests - it's already there. (This does introduce parallelism for the example applications, and the -j option in CI.)

But yes, maybe instead of this change we should explicitly disable all parallelism for all tests.

Reply:
+1 to disable paralleisam to test run tests.